src/ui/common/attributes/AttributeInfo.js

```diff
@@ -9,7 +9,7 @@ import FormLabel from 'ui/common/form/FormLabel';
 import SwitchRenderer from 'ui/common/form/SwitchRenderer';
 import { MODE_EDIT, MODE_ADD } from 'state/data-types/const';
 
-const maxLength10 = maxLength(10);
+const maxLength15 = maxLength(15);
 const maxLength50 = maxLength(50);
```

Comment:
changing this value seems have a global impact to other part of the code that use `AttributeInfo`. Are you sure that this doesn't cause bug in other sections?